bitnami/memcached/README.md

```diff
@@ -78,19 +78,19 @@ The command removes all the Kubernetes components associated with the chart and
 | `image.debug`                                 | Enable image debug mode                                                                         | `false`                      |
 | `architecture`                                | Memcached architecture. Allowed values: standalone or high-availability                         | `standalone`                 |
 | `hostAliases`                                 | Add deployment host aliases                                                                     | `[]`                         |
-| `memcachedUsername`                           | Memcached admin user                                                                            | `nil`                        |
-| `memcachedPassword`                           | Memcached admin password                                                                        | `nil`                        |
+| `memcachedUsername`                           | Memcached admin user                                                                            | `""`                         |
+| `memcachedPassword`                           | Memcached admin password                                                                        | `""`                         |
 | `replicaCount`                                | Number of containers                                                                            | `1`                          |
 | `command`                                     | Default container command (useful when using custom images)                                     | `[]`                         |
 | `arguments`                                   | Default container args (useful when using custom images)                                        | `[]`                         |
 | `extraEnv`                                    | Additional env vars to pass                                                                     | `[]`                         |
 | `podDisruptionBudget.create`                  | Specifies whether a Pod disruption budget should be created                                     | `false`                      |
-| `podDisruptionBudget.minAvailable`            | Minimum number of pods that need to be available                                                | `nil`                        |
+| `podDisruptionBudget.minAvailable`            | Minimum number of pods that need to be available                                                | `""`                         |
```

Comment:
This actually should be an integer, is it something we can define in our metadata?

Reply:
hi @javsalgar 

The possible empty values that we have in readme-generator are "", {} or []. Other option is set up this value to 0.

https://github.com/bitnami/charts/pull/6976#issuecomment-881399315
https://github.com/bitnami/charts/pull/6976#issuecomment-881478391

Right now, the 0 is like a nil but I think that this is not a good practice. If for any reason, they change this approach (0 is nil) we are going to have a 0 instead of nil.

What do you think?

Álvaro

Reply:
I think adding a `0` to `minAvailable` or `maxAvailable` when it is not going to be used can be confused for users. Although it's true those parameters are integers instead of a string, adding a value that is not true and the disable it in the template by adding some logic it's something weird.

The _values.yaml_ may be less readable for users, in the same way, the auto-generated table from the README is going to show `0` as the value but this is not totally true since this `0` is going to act as a "false" boolean.

My 2cts are continuing using an empty string as default parameter and re-evaluate this topic with the team, if needed we can do a batch replacement with `sed` once supported integers in the readme-generator, put a `0` or whatever is the decided approach.